githook.js

```diff
@@ -53,9 +53,11 @@ function main(method, path, query) {
 
   try {
     var remote = data.remote;
-    remote_transforms.forEach(function (transform) {
-      remote = remote.replace(transform.regexp, transform.replacement);
-    });
+    if (data.apply_remote_transform || true) {
```

Comment:
The condition will always be true. Remove the `|| true` perhaps?

Reply:
My javascript is bad, I wanted to make it so that if apply_remote_transform dosn't exist we should still run the transform.

Will re implement it.

Reply:
In modern JS `if (data.apply_remote_transform ?? true)` would accomplish that, I suppose. Unfortunately, I'm not so sure you can use that modern JS in this context.

Reply:
I implemented it the way you suggested. That is much cleaner then '??'